tools/clang/test/CodeGenSPIRV/ternary-op.short-circuited-cond-op.sampler-state.hlsl

```diff
@@ -0,0 +1,10 @@
+// RUN: %dxc -T ps_6_0 -E main -HV 2021 -fcgl  %s -spirv | FileCheck %s
+
+SamplerState sampler_a;
+SamplerState sampler_b;
+Texture2D tex;
+bool selector;
+
+float4 main() : SV_Target0 {
+  return tex.Sample(selector ? sampler_a : sampler_b, (float2)0);
```

Comment:
`FileCheck` will expect some `// CHECK:` comments to verify; you can either remove FileCheck from the RUN line above or add

```suggestion
// CHECK: error: casting to type 'SamplerState' unimplemented
  return tex.Sample(selector ? sampler_a : sampler_b, (float2)0);
```